tox.ini

```diff
@@ -4,14 +4,9 @@ skipsdist = true
 
 [testenv]
 passenv = PYTEST_ADDOPTS
-deps =
-    beautifulsoup4==4.5.1
-    pytest~=3.0.0
-    pytest-base-url==1.1.0
-    pytest-xdist==1.15.0
-    requests==2.11.1
+deps = -rrequirements/tests.txt
```

Comment:
Yes, this is Tox's syntax, and is documented [here](http://tox.readthedocs.io/en/latest/example/basic.html#depending-on-requirements-txt).